helper/schema/schema.go

```diff
@@ -201,6 +201,23 @@ func (s *Schema) GoString() string {
 	return fmt.Sprintf("*%#v", *s)
 }
 
+// Returns a default value for this schema by either reading Default or
+// evaluating DefaultFunc. If neither of these are defined, returns nil.
+func (s *Schema) GetDefaultValue() (interface{}, error) {
+	// Easier to just preload Default even if it's nil than to do another
+	// conditional here
+	defaultValue := s.Default
```

Comment:
I'm not sure if we want this, because the caller needs to be able to distinguish between "no default is set" and "default set to false", so a nil return value has important meaning.